ci-operator/step-registry/openshift-observability/enable-cluster-logging/openshift-observability-enable-cluster-logging-commands.sh

```diff
@@ -0,0 +1,605 @@
+        --from-literal=endpoint="https://minio-service.${MINIO_NAMESPACE}.svc:9000" \
+        --from-literal=access_key_id="minio" --from-literal=access_key_secret="${minio_secret}"
+    schema_version="v13"
+    if [[ "$lo_csv" =~ loki-operator.v5 ]];then
+        schema_version="v12"
+    fi
+
+cat <<EOF|oc create -f -
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: ${LOKI_NAME}
+  namespace: ${LOKI_NAMESPACE}
+spec:
+  managementState: Managed
+  size: ${SIZE}
+  storage:
+    schemas:
+      - effectiveDate: '2023-10-15'
+        version: ${schema_version}
+    secret:
+      name: s3-secret
+      type: s3
+  storageClassName: ${storage_class}
+  tenants:
+    mode: openshift-logging
+  rules:
+    enabled: true
+    selector:
+      matchLabels:
+        openshift.io/log-alerting: 'true'
+    namespaceSelector:
+      matchLabels:
+        openshift.io/log-alerting: 'true'
+EOF
+}
+
+function wait_lokistack_ready(){
+    echo "### Wait the lokistack ready up to 5 minutes"
+
+    lokistack_ready=false
+    count=1
+    # Only validate the ingester pod herae
+    while [[ $count -lt 10 ]]; do
+        if oc -n ${LOKI_NAMESPACE} wait pod --for=condition=Ready  -l app.kubernetes.io/instance=${LOKI_NAME} ; then
+            lokistack_ready=true
+	    break
+        fi
+        sleep 30s
+        (( count=count + 1 ))
+    done
+
+    if [[ $lokistack_ready == "false" ]]; then
+       echo "Exit, lokistack is not ready in 5 minutes"
+       oc -n ${LOKI_NAMESPACE} get pod
+       oc -n ${LOKI_NAMESPACE} get lokistack ${LOKI_NAME} -o jsonpath='.status'
+       exit 1
+    fi
+}
+
+function enable_logging_uiplugin(){
+    echo "### Enable_logging_uiplugin"
+    coo=$(oc get csv -o name |grep observability-operator)
+    if [[ $coo == "" ]];then
+        echo "Skip LoggingUI -- no observability-operator"
+    else
+        echo "deploy LoggingUI Plugin"
+        cat <<EOF|oc apply -f -
+apiVersion: observability.openshift.io/v1alpha1
+kind: UIPlugin
+metadata:
+  name: logging
+spec:
+  logging:
+    logsLimit: 20
+    lokiStack:
+      name: ${LOKI_NAME}
+      namespace: ${LOKI_NAMESPACE}
+    schema: select
+    timeout: 5m
+  type: Logging
+EOF
+    fi
+
+}
+
+## Gather logs and send to Lokistack
+function enable_clf_5x(){
+    echo "Create clf for cluster-logging:v5.x"
+    input_ref=$(echo $INPUTS|sed 's/,/\n/g'| sed 's/^/    - /')
+    cat <<EOF|oc create -f -
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+metadata:
+  name: "instance"
+  namespace: openshift-logging
+spec:
+  managementState: "Managed"
+  logStore:
+    type: "lokistack"
+    lokistack:
+      name: ${LOKI_NAME}
+  collection:
+    type: "vector"
+  visualization:
+    type: ocp-console
+EOF
+
+    cat <<EOF|oc create -f -
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  pipelines:
+    - name: all-to-default
+      inputRefs:
+${input_ref}
```

Comment:
The default value is `"application,infrastructure"`, it's not `all`.

Reply:
Added audit.  we do need to check the audit logs in UI